js/index.js

```diff
@@ -17,3 +17,125 @@ for (let i = 0; i < skills.length; i++) {
     skill.innerText = skills[i];
     skillsList.appendChild(skill);
 }
+
+
+const messageForm = document.querySelector('form[name="leave_message"]');
+const messageSection = document.querySelector('#messages');
+const messageList = messageSection.querySelector('ul');
+
+messageSection.hide = function () {
+    this.style.display = 'none';
+};
+messageSection.show = function () {
+    this.style.display = '';
+};
+
+Object.defineProperty(messageList, "hasNoMessages", {
+    get: function () {
+        return this.childElementCount === 0
+    }
+});
```

Comment:
I don't know how to make it correct (short && readable ).
At first I wrote something like this:
```
function hideMessageSection() {
            messageSection.style.display = 'none';
};
function showMessageSection() {
            messageSection.style.display = 'none';
};
if (messageList.childElementCount === 0) {
            hideMessageSection();
}
```

But then I thought this would be more readable. I am not sure, what to do
`if (messageList.hasNoMessages) {
        messageSection.hide();
    }`